realm/src/main/java/io/realm/DynamicRealmObject.java

```diff
@@ -279,8 +396,8 @@ public void setBoolean(String fieldName, boolean value) {
     /**
      * Sets the {@code short} value of the given field.
      *
-     * @param fieldName Field name.
-     * @param value Value to insert.
+     * @param fieldName field name.
```

Comment:
Why add the many variants of integers as separate methods?
Did you consider just having one or having an setInteger() maybe with overloaded variants?